ComposeTesting/dependencies.gradle

```diff
@@ -46,9 +46,11 @@ ext {
     // AndroidX dependencies
     appCompatVersion = '1.4.1'
     ktxCoreVersion = '1.7.0'
+    windowVersion = '1.0.0'
     androidxDependencies = [
             appCompat         : "androidx.appcompat:appcompat:$appCompatVersion",
             ktxCore           : "androidx.core:core-ktx:$ktxCoreVersion",
+            window            : "androidx.window:window:$windowVersion",
```

Comment:
Why do we need this dependency?

Reply:
This is for the `FoldingFeature.State` parameter in the simulateFoldingFeature methods - it was previously covered by the windowtest artifact I believe but we removed that dependency because the kotlin lib covers everything related to the `WindowLayoutInfoPublisherRule`